src/llaminator.scss

```diff
@@ -43,6 +43,13 @@ main {
   padding-top: calc(2rem + 64px) !important;
 }
 
+llama-item {
+  margin-bottom: 2rem;
+
+  /** TODO: Use a stack based on flexbox */
```

Comment:
I've taken brief looks at flexbox and grid layout before. I think I found grid to be much more manageable.

Reply:
I'm fairly sure I disagree for a simple stack like this, which shouldn't be more complicated than...

```css
main {
  display: flex;
  flex-direction: column;
  row-gap: 2rem;
}
```

However, it's a TODO because we should try properly :)

https://css-tricks.com/snippets/css/a-guide-to-flexbox/